OpenOversight/app/utils.py

```diff
@@ -14,7 +14,7 @@
 def filter_by_form(form, officer_query):
     if form['name']:
         officer_query = officer_query.filter(
-            Officer.last_name.like('%%{}%%'.format(form['name']))
+            Officer.last_name.like('%%{}%%'.format(form['name'].upper))
```

Comment:
We should make sure the data import scripts we use for ETL actually do a .upper on the strings they throw into the DB if we're going to say "everything gets case-insensitive compares to .upper" in the query.